glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
so
>   ## Create
>  Creates the container: file system, namespaces, cgroups, capabilities.
is some kinds of misleading. Only the network namespace can be created
graciously before starting.